### PR TITLE
Add the updated awesome_classifier.py file

### PR DESCRIPTION
**Pull request by khushboogupta13**

### CODE REVIEW

File: awesome_classifier.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+from torchvision import models
+import torch
+dir(models)
+
+alexnet = models.alexnet(pretrained=True)
+
```
Comment (smaranjitghose) [inline awesome_classifier.py:6]: For python scripts, it is a good practice to put all import statements at the beginning of the script

File: awesome_classifier.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ transforms.Resize(256),                  
+ transforms.CenterCrop(224),                
+ transforms.ToTensor(),                    
+ transforms.Normalize(                      
+ mean=[0.485, 0.456, 0.406],                
+ std=[0.229, 0.224, 0.225]                  
+ )])
+
+#dummy path for the image
+img = '/Users/khushboogupta/Desktop/dos.jpg'
```
Comment (smaranjitghose) [inline awesome_classifier.py:20]: You can take it as an input from the user using the command line arguments

File: awesome_classifier.py [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+alexnet.eval()
+
+out = alexnet(batch_t)
+with open('imagenet_classes.txt') as f:
+  classes = [line.strip() for line in f.readlines()]
+_, index = torch.max(out, 1)
+
+percentage = torch.nn.functional.softmax(out, dim=1)[0] * 100
+#printing the label along with the accuracy.
+print(labels[index[0]], percentage[index[0]].item())
```
Comment (smaranjitghose) [inline awesome_classifier.py:34]: For custom models you are expected to store the labels in a list or a dictionary (as a global variable)

File: awesome_classifier.py [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+#evaluation
+alexnet.eval()
+
+out = alexnet(batch_t)
+with open('imagenet_classes.txt') as f:
+  classes = [line.strip() for line in f.readlines()]
+_, index = torch.max(out, 1)
+
+percentage = torch.nn.functional.softmax(out, dim=1)[0] * 100
```
Comment (smaranjitghose) [inline awesome_classifier.py:32]: Out is a very poor choice for the model variable name

File: awesome_classifier.py [hunk 5]
```diff
@@ -0,0 +1,35 @@
+from torchvision import models
+import torch
+dir(models)
+
+alexnet = models.alexnet(pretrained=True)
```
Comment (smaranjitghose) [inline awesome_classifier.py:5]: Next time try to load a custom trained model.

File: awesome_classifier.py [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+from torchvision import transforms
+
+#changing the dimensions of the image.
```
Comment (smaranjitghose) [inline awesome_classifier.py:9]: This can be declared inside in a function. Remember Modularity is a great factor in making software more readable and manageable

File: awesome_classifier.py [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+#dummy path for the image
+img = '/Users/khushboogupta/Desktop/dos.jpg'
+img_t = transform(img)
+batch_t = torch.unsqueeze(img_t, 0)
+
+#evaluation
+alexnet.eval()
+
+out = alexnet(batch_t)
+with open('imagenet_classes.txt') as f:
```
Comment (smaranjitghose) [inline awesome_classifier.py:28]: This is not required. Usually we store the labels in side a list or dictionary or import a json for it

**Review by smaranjitghose — APPROVED** [review]
